nextflow_schema.json

```diff
@@ -706,6 +706,148 @@
             },
             "fa_icon": "fas fa-clone"
         },
+        "damage_manipulation": {
+            "title": "Damage Manipulation",
+            "type": "object",
+            "description": "Options for filtering for, trimming or rescaling characteristic ancient DNA damage patterns",
+            "default": "",
+            "fa_icon": "fas fa-chart-line",
+            "properties": {
+                "run_mapdamage_rescaling": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-map",
+                    "description": "Turn on damage rescaling of BAM files using mapDamage2 to probabilistically remove damage.",
+                    "help_text": "Turns on mapDamage2's BAM rescaling functionality. This probabilistically replaces Ts back to Cs depending on the likelihood this reference-mismatch was originally caused by damage. If the library is specified to be single stranded, this will automatically use the --single-stranded mode.\nThis process will ameliorate the effects of aDNA damage, but also increase reference-bias. \n\n**This functionality does not have any MultiQC output.**\nwarning: rescaled libraries will not be merged with non-scaled libraries of the same sample for downstream genotyping, as the model may be different for each library. If you wish to merge these, please do this manually and re-run nf-core/eager using the merged BAMs as input.\n\n> Modifies the `--rescale` parameter of mapDamage2"
```

Comment:
Use emoji for warnoing i.e., ⚠️ and in a quote block

Reply:
I would maybe mentiopn the MultiQC thing in the output as well, as well as the merging

Reply:
Also for all params, please use the new bullet point style of mentioning which parameters eager is using, see e.g.  `metagenomics_prinseq_mode` for example. This will help e.g. particulary for pmdtools where you have args 1/2/3

Reply:
I gave up on that fight now, can be ignorede